### DISCUSSION TITLE
Welcome to libgpiod Discussions!

### DISCUSSION THREAD

**Discussion by brgl** [brgl/libgpiod#46] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by HankB** [comment]
Hobbyist dev here - (Retired pro - did lots of work with embedded systems.) And I like to play with Raspberry Pis.

I did some exploration of GPIOD here: https://github.com/HankB/GPIOD_Debian_Raspberry_Pi

best,

I'm back! I tried building some C code that uses libgpiod on RPiOS/Trixie and that has now included V2.2, which is great! But of course it breaks my builds. I'm working through my "exploration" and quickly determined that I needed to nuke and pave since the API has been entirely reworked. I've been having an issue with event detection but the (recently found) example code works so I'm sure I'll be able to work it out.

Thanks for all of the work that goes into this and in particular, the examples.

**Comment by Maru006** [comment]
Hello, I am new to GPIOD and really appreciate this library.  

I am aiming to make full use of this library with sensor projects.

I will be posting all my code in this public repo https://github.com/Maru006/contrib.git

Recently unlocked DHT11 humidity and Sensors!
